WoWDatabaseEditor.Common/WDE.DbcStore/DbcStore.cs

```diff
@@ -166,7 +185,7 @@ private void Load(string filename, int id, int nameIndex, Dictionary<long, strin
             
             private void Load(string filename, int id, int nameIndex, Dictionary<long, long> dictionary)
             {
-                Load(filename, row => dictionary.Add(row.GetInt(id), row.GetInt(nameIndex)));
+                Load(filename, row => dictionary.Add(row.GetInt(id), row.GetInt(nameIndex + (int) dbcSettingsProvider.GetSettings().DBCLocale)));
```

Comment:
that makes 0 sense to me

Reply:
it should be used in `Load(string filename, int id, int nameIndex, Dictionary<long, string> dictionary)`, a version which loads a string, not an int **AND** it is only required sometimes (not all strings have locale versions), so I'd recommend adding a bool parameter whether it should take a locale version or not (i.e. `Map` should use locale version, but `MapDirectory` should not)

Reply:
If you want german locale ,load german dbc.
If you want french locale, load french dbc.
etc.

Reply:
> If you want german locale ,load german dbc. If you want french locale, load french dbc. etc.

not really, in the old dark times of Wrath (and cata? not sure here) there were 16 columns per string column for each locale, but only one is filled (others are empty)

Reply:
i use french DBC, and the tool send me an error message when i want to open it. Now with that you can specify directly on DBC option the locale you want to use, and the Loader go into the right column for string localisation